src/main/java/bio/overture/ego/service/UserPermissionService.java

```diff
@@ -52,6 +52,13 @@ public UserPermissionService(
     this.apiKeyEventsPublisher = apiKeyEventsPublisher;
   }
 
+  protected PolicyResponse convertToPolicyResponse(@NonNull UserPermission userPermission) {
```

Comment:
add `@Override`

Reply:
i removed the method from the AbstractPermissionService so there's nothing to override...AbstractNameablePermissionService has its own implementation that uses `owner.getName`